proxywasm/internal/abi_callback_alloc.go

```diff
@@ -14,9 +14,15 @@
 
 package internal
 
-//nolint
+import "time"
+
+// nolint
+//
```

Comment:
(not sure why I put nolint here, could we remove this?)

Reply:
It was because of `unused`. I added a simple test for the function